addons/im_livechat/views/im_livechat_chatbot_templates.xml

```diff
@@ -7,9 +7,7 @@
                 <t t-set="chatbot_name" t-value="chatbot_script.title" />
                 <t t-set="head">
                     <script>
-                        <t t-call="im_livechat.loader">
-                            <t t-set="info" t-value="{ 'available': True, 'options': { 'chatbot_test_store': chatbot_test_store }, 'server_url': server_url }"/>
-                        </t>
+                        <t t-call="im_livechat.loader" info="{ 'available': True, 'options': { 'chatbot_test_store': chatbot_test_store }, 'server_url': server_url }"/>
```

Comment:
Could you split your work in multiple commits so I can be sure which part is what please? :pray: I'll wait to review that last commit :pray: